_data/doi_license.yml

```diff
@@ -4,3 +4,13 @@
   doi: https://zenodo.org/record/1326873#.XA_xnCdRcWo
```

Comment:
@jlpalomino just curious. why is the format of this DOI url different from the ones below?

Reply:
@lwasser I am not sure; I believe you added those DOI urls in PR #715.  Would like me to change this one and the one for earth-analytics-bootcamp to the latest doi badge?

Reply:
thanks @jlpalomino  yes please. i think at the time, i didn't know the difference between the DOI links in zenodo! Please update them in this PR.
Thank you!!

Reply:
@lwasser Done!